### ISSUE TITLE
NullPointerException in FranceSouthWestProvider (Transportr)

### ISSUE THREAD

**Issue by gbaudic** [schildbach/public-transport-enabler#278]
When trying to find an itinerary in Transportr around Toulouse which should involve using TER (regional train), the following stack trace shows up:

```
java.lang.NullPointerException
com.google.common.base.Preconditions.checkNotNull(Preconditions.java:890)
de.schildbach.pte.dto.Style.parseColor(Style.java:108)
de.schildbach.pte.FranceSouthWestProvider.getLineStyle(FranceSouthWestProvider.java:57) 
```

This line corresponds in the code to SUBURBAN_TRAIN, a category which as far as I know should not be used in this region. According to the comments in the code, TER train service should show up as REGIONAL_TRAIN.

**Comment by gbaudic** [comment]
I just noticed that this may well be addressed by PR #169 .

**Comment by schildbach** [comment]
Should be fixed via 7f607160ca919bd68628d867f0a80972c7a471ae.
